packages/babel-plugin-transform-react-constant-elements/test/fixtures/integration/issue-10879/output.js

```diff
@@ -4,19 +4,17 @@ var _react = babelHelpers.interopRequireDefault(require("react"));
 
 var _jsxRuntime = require("react/jsx-runtime");
 
-var namespace = {
-  MyComponent: function MyComponent(props) {
-    return props.name;
-  }
+const namespace = {
+  MyComponent: props => props.name
 };
 
-var buildTest = function buildTest(name) {
+const buildTest = name => {
```

Comment:
We already have `issue-10879-babel-7` for this test, so good to overwrite the test output.